ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/byo-fw/provision/cucushift-installer-rehearse-gcp-ipi-byo-fw-provision-chain.yaml

```diff
@@ -0,0 +1,25 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-byo-fw-provision
+  steps: 
+    - ref: gcp-provision-minimal-permission
+    - ref: gcp-provision-vpc
+    - ref: gcp-provision-firewalls
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-gcp-firewall-rules-management
+    - ref: single-node-conf-gcp
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
```

Comment:
when using byo-fw, will installer tag the user provided firewall rules with certain labels(such as, `shared` tags)? or do we need some additional checks on the pre-created firewall rules to make sure installer doesn't modify them.

Reply:
Good question! But in fact, the given GCP credential to the installer doesn't have the permission, anyway, let me think about it. Thanks!